src/components/App.js

```diff
@@ -1,14 +1,14 @@
 import React, { useState, useEffect } from 'react';
 
 import {
-  getSomething
+  getProducts
 } from '../api';
 
 const App = () => {
   const [message, setMessage] = useState('');
 
   useEffect(() => {
-    getSomething()
+    getProducts()
       .then(response => {
         setMessage(response.message);
       })
```

Comment:
Do we need an empty array at the end of useEffect() prior to the close of the () on line 14?